### DISCUSSION TITLE
blog/wordle-guess-helper/

### DISCUSSION THREAD

**Discussion by giscus** [gadenbuie/garrickadenbuie-com#110] [Comments]
# blog/wordle-guess-helper/

Picking words to guess in Wordle.
It&rsquo;s only fun if you can solve it with R.

https://www.garrickadenbuie.com/blog/wordle-guess-helper/

**Comment by Hantale** [comment]
A fun little calculator! However, I do have one issue with the list used for recognizing entropic value, and that is it seems to only include letters that are confirmed or available, which seems like an oversimplification.

EG: AROSE, lets say all but the O is grey, and the O is green. Every word generated by the bot to try next will have an O as the middle letter.

But unless you have a good chance of getting it on the second round, you're strategically better off guessing with 5 new letters! You already know the O is there, so repeating it on the next guess gives you 9 letters of information instead of 10.

**Reply by gadenbuie** [reply]
This is a great point! I guess technically this is a "hard mode" calculator. I noticed this too and decided to keep myself from failing into an endless rabbit hole of scope creep with this disclaimer:

> It is not the best strategy for Wordle or even the best way to play. But along the way we’ll learn a few text processing tricks, we’ll write a few functions, and we’ll learn how to move seamlessly from R to the browser in the same document or blog post.

**Comment by Hantale** [comment]
Its definitely a fun practice example at the very least!

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by jigsaw300puzzles** [comment]
This <a href="https://wordlesvenska.com/">Wordle</a> helper strikes a nice balance—it gives you useful guidance without spoiling the fun. Perfect for improving your strategy while keeping the challenge intact!
